src/features/FEATURES-overview.md

```diff
@@ -0,0 +1,18 @@
+
+# struktura
```

Comment:
Podla mna nepatri `.md` do `src/` lebo to nie je zdrojak appky. Ja by som to dal kludne ako sekiu do README. Ak nie, tak by som pridal odkaz do hlavneho README aby sa to dalo lahsie najst.

A tiez by som zjednotil jazyk dokumentacie - bud cele anglicky alebo cele slovensky :)

Reply:
.md je readme akehokolvek druhu ktore sa tyka akehokolvek foldru, mal by byt v zasade v kazdom foldri a vysvetlvoat co sa tam nachadza ako onboarding

v roote je strasne vela suborov kde sa to strati, vo features zase je super ze prides a hned tam mas intro ze co tam najdes

davat to do hlavneho readme (ktore je uz aj tak moc dlhe) by sposobilo ze utrzok ktory ma samostatne vyuzitie sa spaja do suboru kde je vela inych veci - kod ma vyuzivat separation of concern, vsetko co sa da samostatne navnimat by malo byt samostatne (do miery kym to nie je otravne sa prepinat)

jazyk dokumentacie podla mna je uplne jedno, som ok prepisat hocico do hocicoho, ale proste niektore veci sa lahsie hovoria slovensky niektore anglicky, normalne som za anglictinu, ale vzhladom na to ze to je slovensky projekt, tak je to jedno, a nevidim problem dat full do jedneho jazyka, staci ukazat kde to je problem a mozes sa do toho pustit

Reply:
ked sa zhodnu viaceri ze je to blocker

Reply:
> davat to do hlavneho readme (ktore je uz aj tak moc dlhe)

Tak potom aspon link z hlavneho README. To je miesto kde vacsina ludi zacina :)

Reply:
its up to you, pre mna to je skor pomocka na onboarding developerov, kludne sa to moze schovat niekam hlbsie, ale bude to tam vzdy dostupne na odporucanie, a budu tam pribudat quickstart info, tak aby tam nikdy neboli informacie ktore na onboardnutie nepotrebujes vediet

Reply:
podla mna ked vojdes do foldru features, tak to tam zbadas, a ak budes mat otazku ze co to tam vidis tie foldre, tak ti "features overview" pritiahne zrak, ak nie tak je to fajn a nemusis to riesit

ci to dat do hlavneho readme, nemam nazor, ale v zasade skor by som ocenil v kazdom feature foldri nejake intro ze o co ide, a proste to je dokumentacia kontextu toho kodu, a readme je pre mna skor rozcestnik vsetkeho, takze up to anyone ci to tam niekto chce mat, nie som proti ani za